python/dgl/nodeflow.py

```diff
@@ -293,6 +335,7 @@ def map_to_parent_nid(self, nid):
             The parent node id array.
         """
         nid = utils.toindex(nid)
+        # TODO(minjie): should not directly use []
```

Comment:
why?

Reply:
Because this syntax doesn't work with Chainer variables.  I fixed these in my Chainer PR.